src/Controls/src/Core/Handlers/Items/Android/MauiCarouselRecyclerView.cs

```diff
@@ -571,46 +571,42 @@ void ClearLayoutListener()
 
 		protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 		{
-			if (Carousel.Loop)
+			// If the height or width are unbounded and the user is set to
```

Comment:
I only removed the `if (Carousel.Loop)` condition but indentation changes make it harder to spot